ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake

```diff
@@ -21,9 +21,10 @@
 # @public
 #
 macro(ament_lint_auto_find_test_dependencies)
-  if(ARGN)
+  set(_ARGN "${ARGN}")
+  if(_ARGN)
     message(FATAL_ERROR "ament_lint_auto_find_test_dependencies() called with "
-      "unused arguments: ${ARGN}")
+      "unused arguments: ${_ARGN}")
```

Comment:
These changes look unrelated. Why did you change them?

Reply:
That's related to this comment: https://github.com/ament/ament_lint/pull/133#discussion_r268343524

I'll submit them separately - this PR originally had a second change to these lines where it checked ARG_UNPARSED_ARGUMENTS

Reply:
https://github.com/ament/ament_lint/pull/136